.bazelrc

```diff
@@ -118,7 +118,7 @@ build:linux --config=dynamic_kernels
 # For projects which use TensorFlow as part of a Bazel build process, putting
 # nothing in a bazelrc will default to a monolithic build. The following line
 # opts in to modular op registration support by default.
-build --define framework_shared_object=true
+build --define framework_shared_object=false
```

Comment:
Can you update the comment above too since you're changing this value?

Reply:
Done.